project/src/main/webapp/candidate.html

```diff
@@ -19,7 +19,7 @@ <h2>Voter Central Candidate Page</h2>
       <div class="candidate-page-info" id="official-info-container">
       </div>
       <div class="candidate-page-info" id="news-articles-container">
-        <article id="news-article-1">
+        <!-- <article id="news-article-1">
```

Comment:
Do you plan on uncommenting these in the future? If so, add a TODO for when it will be appropriate to do so e.g. "This block of code should be uncommented after implementing xyz"